pkg/config/root_config.go

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import (
+	"github.com/seata/seata-go/pkg/common/log"
+	"go.uber.org/atomic"
+	"sync"
+)
+
+var (
+	startOnce sync.Once
+	exporting = &atomic.Bool{}
+)
+
+// RootConfig is the root config
+type RootConfig struct {
+	PrefixStr     string           `default:"seata"`
+	ServerConf    *ServerConfig    `yaml:"server" json:"server" property:"server" hcl:"server"`
+	StoreConf     *StoreConfig     `yaml:"store" json:"store" property:"store" hcl:"store"`
+	RegistryConf  *RegistryConfig  `yaml:"registry" json:"registry" property:"registry" hcl:"registry"`
+	MetricsConf   *MetricsConfig   `yaml:"metrics" json:"metrics" property:"metrics" hcl:"metrics"`
+	TransportConf *TransportConfig `yaml:"transport" json:"transport" property:"transport" hcl:"transport"`
```

Comment:
why RootConfig not contains other config, such as gettyConfig, profilesConfig......